metasynth/distribution/base.py

```diff
@@ -35,6 +35,8 @@ def fit(cls, series: Sequence, *args, **kwargs) -> BaseDistribution:
             Fitted distribution.
         """
         pd_series = cls._to_series(series)
+        if len(pd_series) == 0:
+            return cls._example_distribution()
```

Comment:
Ok nice; 

I notice we are using example distribution a lot throughout, should we call it something different? Default distribution?

Reply:
> Ok nice;
> 
> I notice we are using example distribution a lot throughout, should we call it something different? Default distribution?

Yes, I wasn't planning to use it that much, but now that it's usage has increased, maybe a `default_distribution` makes more sense. Let's do it in another pull request indeed.